packages/query-core/src/tests/hydration.test.tsx

```diff
@@ -467,3 +467,4 @@ describe('dehydration and rehydration', () => {
     expect(hydrationCache.find(['string'])?.state.fetchStatus).toBe('idle')
   })
 })
+console.log('here')
```

Comment:
```suggestion

```

Reply:
🤦‍♂️ I'm sure this is me testing the cache-busting and accidentally commiting the change lol